### ISSUE TITLE
Speaker Photo Alt Text Not Sufficiently Descriptive

### ISSUE THREAD

**Issue by dzoladz** [code4lib/2022.code4lib.org#4]
At the moment, the alt text for any speaker photo only provides the speaker's full name. Alt text for speaker photos would ideally be actual descriptions of the speakers, not just a full name.

From @metageeky 
> The main goal is that without descriptions of the speakers, a screen reader user does not know about the perceived gender, potential race/ethnicity, etc. of the speakers except by judging names alone. This is a broader diversity issue. Of course, we cannot presume gender or race unless told. There are ways to describe people without saying race or gender explicitly. That's part of the art of alt text.

We'll likely need to:

1. Add another data point in the speaker objects found in `_data/examples/speakers.yml` to record the alt text description.
2. Update `_includes/speaker_box.html` to pull in the new alt text description from the speaker object, rather than `alt="{{ speaker.name }}` as it currently does

**Comment by kaitlinnewson** [comment]
A question for this issue is how we should handle missing photos and missing alt-text. I can bring this to the accessibility committee when we meet too.

For missing alt text, should we default to the speaker name, or an empty alt attribute?

For a missing photo, should we add alt-text to the "no pic" photos? e.g. the photos of a pug with 404 text.

**Comment by dzoladz** [comment]
Following the merge of #4, moving the conversation to #5 and closing this one.
